src/Payload.cpp

```diff
@@ -43,6 +43,13 @@ std::string Payload::moveDataToString() {
   return data->moveToFbString().toStdString();
 }
 
+std::string Payload::cloneDataToString() const {
```

Comment:
By the way, I did not mean we should not have the `<<` overload. That would be great to have.